backend/src/main/java/org/ukma/spring/crooodle/entities/HotelEntity.java

```diff
@@ -9,35 +9,44 @@
 import org.hibernate.validator.constraints.Length;
 import org.ukma.spring.crooodle.entities.embedded.CoordinatesEntity;
 
+import java.util.List;
+
 @AllArgsConstructor
 @NoArgsConstructor
 @Builder
 @Data
 @Entity
 @Table(name = "hotels")
 public class HotelEntity {
+
     @Id
     @GeneratedValue(strategy = GenerationType.IDENTITY)
     private Long id;
 
     @Length(min = 1, max = 500)
     @Column(nullable = false)
     private String name;
+
     @Length(min = 1, max = 500)
     @Column(nullable = false)
     private String address;
 
     @PositiveOrZero
     private long rankSum;
+
     @PositiveOrZero
     private int rankCount;
 
     @Embedded
     private CoordinatesEntity coordinates;
 
     @ManyToOne
-    private CountryRegionEntity country_region;
+    private WorldRegionEntity country_region;
```

Comment:
Country and world region are not the same. The hotel isn't located in "Asia" for example